test/compilable/extra-files/ddoc17697.html

```diff
@@ -1,484 +1,506 @@
-
```

Comment:
Please either do the newline change in its own commit, or simply commit this file with the same line endings it uses currently.